### ISSUE TITLE
Can't load the 'dictionary' source

### ISSUE THREAD

**Issue by ktrNozomi** [uga-rosa/cmp-dictionary#40]
The Neovim version:

![image](https://user-images.githubusercontent.com/86712470/219034909-3a336cff-97cd-45a4-b831-08beccb1fb40.png)

Other sources work normally and `'dictionary'` can't work:

![image](https://user-images.githubusercontent.com/86712470/219035158-4aff0c3e-642e-447a-a5b7-76a455d1a693.png)

Here's my settings:

![image](https://user-images.githubusercontent.com/86712470/219035718-6a3e107f-0880-453b-be79-d63d8ad02719.png)

![image](https://user-images.githubusercontent.com/86712470/219035623-3eedd6a2-ae5f-42e6-9557-b99ef9e3984a.png)

The `.dic` file is downloaded from github, I don't know if it would be the cause.

**Comment by ktrNozomi** [comment]
![image](https://user-images.githubusercontent.com/86712470/219036988-04533a38-2721-44e6-9b03-fb10755ddba9.png)

I have changed `dict` to `dic` but still does not work.

**Comment by uga-rosa** [comment]
Yes. I have made a destructive change and released v2.0.0. The new setup instructions are in the doc; you can also use the previous version with a tag designation.

**Comment by Dosx001** [comment]
@uga-rosa I'm having trouble with the new release. Not sure what I'm doing wrong.

old config
```lua
require("cmp_dictionary").setup({
  dic = {
    ["*"] = "~/en.dict",
  },
})
```
new config
```lua
require("cmp_dictionary").switcher({
  spelllang = {
    en = "~/en.dict",
  },
})
```
